packages/aztecoo/src/Epetra_MsrMatrix.h

```diff
@@ -413,6 +413,6 @@ class Epetra_MsrMatrix: public Epetra_Object, public Epetra_CompObject, public v
     
 
  //! Copy constructor (not accessible to users).
-  Epetra_MsrMatrix(const Epetra_MsrMatrix & Matrix) {(void)Matrix;}
+  Epetra_MsrMatrix(const Epetra_MsrMatrix & Matrix) : Epetra_Object(Matrix), Epetra_CompObject(Matrix) {(void)Matrix;}
```

Comment:
Does the body of the constructor actually need to reference `Matrix`, if the parent class constructors do?

Reply:
Sorry @mhoemmen, I'm not sure what exactly you're asking.

Reply:
Does the following work and not emit build warnings?
```c++
Epetra_MsrMatrix(const Epetra_MsrMatrix & Matrix) : 
  Epetra_Object(Matrix), 
  Epetra_CompObject(Matrix)
{}
```

Reply:
Yup, looks like that works just fine.